waku-bindings/src/node/filter.rs

```diff
@@ -28,7 +28,7 @@ pub fn waku_filter_subscribe(
         .into_raw();
 
     let result_ptr = unsafe {
-        let result_ptr = waku_sys::waku_filter_subscribe(
+        let result_ptr = waku_sys::waku_legacy_filter_subscribe(
```

Comment:
Why is this legacy now? And what is the newest call?

Reply:
The newest calls can be seen here: https://github.com/waku-org/go-waku/blob/master/library/README.md#waku-filter . These run on a protocol different from filterv1 and are not compatible. Eventually these filter functions should be deprecated and new filter functions using filterv2 must be added.

Reply:
Specs for filterV2 can be read here: https://rfc.vac.dev/spec/12/